MC/config/PWGHF/pythia8/generator/pythia8_beautyhadronic_with_decays_Mode2.cfg

```diff
@@ -0,0 +1,174 @@
+4232:addChannel = 1 0.2 0 3324 211
+4232:addChannel = 1 0.2 0 3312 211 211
+### add Xic0 decays absent in PYTHIA8 decay table
+4132:addChannel = 1 0.0143 0 3312 211
+### add OmegaC decays absent in PYTHIA8 decay table
+4332:addChannel = 1 0.5 0 3334 211
+4332:addChannel = 1 0.5 0 3312 211
+
+### K* -> K pi
+313:onMode = off
+313:onIfAll = 321 211
+### for Ds -> Phi pi+
+333:onMode = off
+333:onIfAll = 321 321
+### for D0 -> rho0 pi+ k-
+113:onMode = off
+113:onIfAll = 211 211
+### for Lambda_c -> Delta++ K-
+2224:onMode = off
+2224:onIfAll = 2212 211
+### for Lambda_c -> Lambda(1520) K-
+102134:onMode = off
+102134:onIfAll = 2212 321
+### for Xic0 -> pi Xi -> pi pi Lambda -> pi pi pi p 
+### and Omega_c -> pi Xi -> pi pi Lambda -> pi pi pi p
+3312:onMode = off
+3312:onIfAll = 3122 -211
+3122:onMode = off
+3122:onIfAll = 2212 -211
+### for Omega_c -> pi Omega -> pi K Lambda -> pi K pi p
+3334:onMode = off
+3334:onIfAll = 3122 -321
+
+### switch off all decay channels
+411:onMode = off
+421:onMode = off
+431:onMode = off
+4112:onMode = off
+4122:onMode = off
+4232:onMode = off
+4132:onMode = off
+443:onMode = off
+4332:onMode = off
+511:onMode = off
+521:onMode = off
+531:onMode = off
+5122:onMode = off
+
+### D0 -> K pi
+421:onIfMatch = 321 211
+
+### D+/- -> K pi pi 
+411:onIfMatch = 321 211 211
+### D+/- -> K* pi
+411:onIfMatch = 313 211
+### D+/- -> phi pi
+411:onIfMatch = 333 211
+
+### D_s -> K K*
+431:onIfMatch = 321 313
+### D_s -> Phi pi
+431:onIfMatch = 333 211
+
+### Lambda_c -> p K*
+4122:onIfMatch = 2212 313
+### Lambda_c -> Delta K
+4122:onIfMatch = 2224 321
+### Lambda_c -> Lambda(1520) pi
+4122:onIfMatch = 102134 211
+### Lambda_c -> p K pi
+4122:onIfMatch = 2212 321 211
+### Lambda_c -> pK0s
+4122:onIfMatch = 2212 311 
+
+### Xic+ -> pK*0
+4232:onIfMatch = 2212 313
+### Xic+ -> p K- pi+
+4232:onIfMatch = 2212 321 211
+### Xic+ -> Xi*0 pi+, Xi*->Xi- pi+
+4232:onIfMatch = 3324 211
+### Xic+ -> Xi- pi+ pi+
+4232:onIfMatch = 3312 211 211
+    
+### Xic0 -> Xi- pi+
+4132:onIfMatch = 3312 211
+
+### Omega_c -> Omega pi
+4332:onIfMatch = 3334 211
+### Omega_c -> Xi pi
+4332:onIfMatch = 3312 211
+
+### Force also golden beauty hadrons decay modes for D2H studies
+### add B0 decays
+511:oneChannel = 1 0.4 0 -411 211
+511:addChannel = 1 0.3 0 -413 211
+511:addChannel = 1 0.2 0 431 -211
+### add B+ decays
+521:oneChannel = 1 0.8 0 -421 211
+### add Bs0 decays
+531:oneChannel = 1 0.8 0 -431 211
+### add Lb decays
+5122:oneChannel = 1 0.8 0 4122 -211
+
+### we also add channels useful for study of partly reconstructed decays / correlated backgrounds
+### add B0 decays
+511:oneChannel = 1 0.1 0 -411 213
+### add B+ decays
+521:oneChannel = 1 0.1 0 -421 213
+521:oneChannel = 1 0.1 0 -423 211
+### add Bs0 decays
+531:oneChannel = 1 0.1 0 -431 213
+531:oneChannel = 1 0.1 0 -433 211
+### add Lb decays
+5122:oneChannel = 1 0.1 0 4122 -213
+5122:oneChannel = 1 0.05 0 4122 211 -211 -211
+5122:oneChannel = 1 0.05 0 4212 -211
```

Comment:
Hi @fgrosa ! Silly question, but why don't we set `onIfmatch` commands for beauty hadrons, as we did [here](https://github.com/AliceO2Group/O2DPG/blob/master/MC/config/PWGHF/pythia8/generator/pythia8_charmtriggers_with_Bforced_decay.cfg#L118-L132)?